Makefile

```diff
@@ -62,7 +62,7 @@ push:
 		docker push $(REGISTRY)/debian-venti:go$$goversion-$(DEBIAN_VERSION) ; \
 	done
 	docker tag debian-venti:$(DEBIAN_VERSION) $(REGISTRY)/debian-venti:$(DEBIAN_VERSION)
-	for version in 0.0.3 $(DEBIAN_VERSION); do \
+	for version in 0.0.3 latest $(DEBIAN_VERSION); do \
```

Comment:
Pretty sure most of the docker community considers `latest` to be an anti-pattern. I'm not opposed to it, depending on how it get's used though.

Reply:
I agree, lets leave `latest` out.  In general, I want to move our use & publishing of these images towards determinism -- so that builds are reproduceable.  Latest works against that.